doc/server.js

```diff
@@ -3,6 +3,6 @@ const express = require('express')
 const app = express()
 app.use(express.static('./build/site'))
 
-app.get('/', (req, res) => res.redirect('/neo4j-streams-docs/4.0.2'))
+app.get('/', (req, res) => res.redirect('/kafka/4'))
```

Comment:
Correct! Because this URL must match the following format:

```
/<Antora playbook name>/<Antora playbook version>
```

I didn't realize that `name` and `version` were changed by Adam in my PR.

Reply:
@moxious Could you please fix the URL as follow?

```
app.get('/', (req, res) => res.redirect('/kafka/4.0'))
```

Thanks!